config/config-feature-flags.yaml

```diff
@@ -131,3 +131,5 @@ data:
   enable-artifacts: "false"
   # Setting this flag to "true" will enable the built-in param input validation via param enum.
   enable-param-enum: "false"
+  # Setting this flag to "true" will enable the use of concise resolver syntax
```

Comment:
Do we document stability levels for feature flags anywhere like https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features?

Reply:
Yes. https://tekton.dev/docs/installation/additional-configs/#alpha-features

Thanks for pointing that out. I need to update that table.

Reply:
I think I'll have to do that in a follow-up PR after the release so that I have the release link etc.